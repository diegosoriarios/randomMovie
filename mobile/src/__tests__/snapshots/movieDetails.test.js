import React from 'react';

import renderer, { act } from 'react-test-renderer';
import MovieDetail from '../../pages/movieDetail/index';

jest.mock('@react-navigation/native');

test('renders correctly', async (done) => {
  const route = {
    params: {
      movie: {
        vote_average: '',
        image: '',
        title: '',
        overview: '',
        genre_ids: '',
      },
    },
  };
  const tree = renderer.create(<MovieDetail route={route} />).toJSON();
  expect(tree).toMatchSnapshot();
  done();
});
