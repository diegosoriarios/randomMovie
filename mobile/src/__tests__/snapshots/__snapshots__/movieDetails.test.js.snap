// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#00b4d8",
        "height": "40%",
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "flex-start",
        "top": 25,
        "width": "100%",
      }
    }
  >
    <View
      accessibilityLabel="backButton"
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "borderRadius": 50,
          "elevation": 20,
          "height": 50,
          "justifyContent": "center",
          "marginLeft": 20,
          "opacity": 1,
          "shadowColor": "#000",
          "shadowOpacity": 1,
          "shadowRadius": 20,
          "width": 50,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 20,
            },
            undefined,
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
  <View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "padding": 20,
        }
      }
    >
      <View
        style={
          Object {
            "justifyContent": "space-evenly",
            "width": "40%",
          }
        }
      >
        <Text>
          
        </Text>
      </View>
      <View
        style={
          Object {
            "width": "60%",
          }
        }
      >
        <Image
          alt=""
          source={
            Object {
              "uri": "",
            }
          }
          style={
            Object {
              "aspectRatio": 1.2,
              "width": "100%",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityLabel="bookmarkButton"
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderRadius": 50,
        "elevation": 20,
        "height": 50,
        "justifyContent": "center",
        "left": 20,
        "marginTop": -15,
        "position": "absolute",
        "shadowColor": "#000",
        "shadowOpacity": 1,
        "shadowRadius": 20,
        "top": "40%",
        "width": 50,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#000",
            "fontSize": 20,
          },
          undefined,
          Object {
            "fontFamily": "Material Icons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      
    </Text>
  </View>
  <View
    style={
      Object {
        "margin": 20,
        "position": "absolute",
        "top": "45%",
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 25,
          "fontWeight": "bold",
        }
      }
    >
      
    </Text>
    <Text
      style={
        Object {
          "fontSize": 18,
          "textAlign": "justify",
        }
      }
    >
      
    </Text>
  </View>
</View>
`;
